.coafile

```diff
@@ -0,0 +1,41 @@
+[Default]
+ignore = .tox/**, .gitignore, .gitreview, .gitmodules, **/provisioning/roles/**
```

Comment:
I added bear for puppet. For now, I'm ignoring class-name related warnings by puppet bear. 

And for Ruby, what Bear should we go for [RuboCopBear](https://github.com/coala/bear-docs/blob/master/docs/RuboCopBear.rst)  or [RubySyntaxBear](https://github.com/coala/bear-docs/blob/master/docs/RubySyntaxBear.rst)? RuboCopBear gives a lot of warnings, a lot like "Method/class has too many lines", line-length, using Ruby 1.9 hash syntax, use of braces etc while RubySyntaxBear just checks for the syntax and has few (5-6) warnings. We can go with the first one, ignore all the warnings for now and follow it in future. @dfarrell07 what do you suggest?